extensions/guacamole-auth-quickconnect/src/main/resources/styles/quickconnect.css

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+.quickconnect-container {
+    margin: 0.5em 0;
+    width: 100%;
+}
+
+.quickconnect-container .quickconnect-field {
+    background-image: url('images/protocol-icons/guac-text.png');
+    background-repeat: no-repeat;
+    background-size: 1.75em;
+    background-position: 0.25em center;
+    background-color: transparent;
+    padding: 0.5em;
+    padding-left: 2.25em;
+    width: 100%;
+    max-width: none;
+    border: 0;
+    border-left: 1px solid rgba(0,0,0,0.125);
+    box-sizing: border-box;
+}
+
+.quickconnect-field input[type="submit"] {
```

Comment:
Is there an `<input type="submit" ...>`? I don't see one in the HTML.

Reply:
Nope.  Removed.